themes/default/content/blog/developer-experience-infrastructure-kubernetes-automation-with-okteto-pulumi/index.md

```diff
@@ -23,7 +23,7 @@ meta_desc: "Platform engineers need to prioritize automating cloud resource prov
 # The meta_image appears in social-media previews and on the blog home page. A
 # placeholder image representing the recommended format, dimensions and aspect
 # ratio has been provided for you.
-meta_image: /images/resources/developer-infra-okteto-engin-arsh.png
+meta_image: https://www.pulumi.com/images/resources/developer-infra-okteto-engin-arsh.png
```

Comment:
we should keep this as the relative location the way it was before

Reply:
Ah, ok. It didn't seem to have fixed the issue, either.